public/ui/cards.js

```diff
@@ -5,14 +5,37 @@ export default class Cards extends Component {
 	render(props) {
 		return html`
 			<div class="Cards">
-				${props.cards.map((card) => Card(card, props.energy))}
+				${props.cards.map(card => Card(card, props.energy, props.cards))}
 			</div>
 		`
 	}
 }
 
-export function Card(card, currentEnergy) {
-	const isDisabled = currentEnergy < card.energy
+function isCardDisabled(card, currentEnergy, cards) {
+	let isDisabled;
```

Comment:
```suggestion
	let isDisabled = false
```

to make it a tiny bit clearer